cyclonedx/model/bom.py

```diff
@@ -90,7 +91,7 @@ def __init__(
                 DeprecationWarning)
 
         if not tools:
-            self.tools.tools.add(ThisTool)
+            self.tools.components.add(this_component())
```

Comment:
W T F ?! 

why is `tols` a  property of bom? it belongs in the metadata ... right?